src/spelling/dictionary_provider_nsspellchecker.mm

```diff
@@ -132,14 +132,19 @@ bool isValid() const
 	NSString* nsstring = [NSString stringWithCharacters:reinterpret_cast<const unichar*>(word.unicode()) length:word.length()];
 
 	NSArray* array;
-	if ([[NSSpellChecker sharedSpellChecker] respondsToSelector:@selector(guessesForWordRange)]) {
-		array = [[NSSpellChecker sharedSpellChecker] guessesForWordRange:range
-			inString:nsstring
-			language:m_language
-			inSpellDocumentWithTag:m_tag];
-	} else {
-		array = [[NSSpellChecker sharedSpellChecker] guessesForWord:nsstring];
-	}
+    //if ([[NSSpellChecker sharedSpellChecker] respondsToSelector:@selector(guessesForWordRange)]) {
+        array = [[NSSpellChecker sharedSpellChecker] guessesForWordRange:range
+            inString:nsstring
+            language:m_language
+            inSpellDocumentWithTag:m_tag];
+
+    /* Disable support for macOS 10.0–10.6 SDK.
```

Comment:
If you are confident that we shouldn't have this check here, then go ahead and remove the commented out code.

Reply:
Yes, We're targeting at least for 10.10 because brew/Qt. This code is pretty old.